test/TestHelper.cpp

```diff
@@ -108,6 +108,12 @@ void ImportTest::importState(json_spirit::mObject& _o, State& _state)
 		BOOST_REQUIRE(o.count("storage") > 0);
 		BOOST_REQUIRE(o.count("code") > 0);
 
+		bigint biValue256 = bigint(1) << 256;
```

Comment:
`bigint biValue256 = bigint(1) << 256;`

becomes:

`bigint const c_max256plus1 = bigint(1) << 256;`

(move descriptive variable name, note it is `const`).